### ISSUE TITLE
Pre-work Checklist: Developer: Alejandro Gomez

### ISSUE THREAD

**Issue by agosmou** [hackforla/website#3787]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@agosmou I have added you to the GitHub repo for hackforla website and to the Google Drive for the project. Please continue on the steps above to continue your journey on the website team (assuming you still want to). Thanks!

**Comment by ExperimentsInHonesty** [comment]
@agosmou I have invited you to all three calendar invites for the hackforla.org website team.

**Comment by ExperimentsInHonesty** [comment]
@agosmou Are you joining us?  If so, please work on the checkboxes above. If not, please let us know so that we can close out this issue.

**Comment by agosmou** [comment]
Hi Bonnie,

I was out of town for a few weeks. I’ll be back in LA tonight. I’ll get
this worked on! Talk soon.

Ale

**Comment by agosmou** [comment]
Hi Bonnie,

I'm all settled in now. I'll get to working on this between today and
tomorrow. :)

Thanks,

Ale

**Comment by ExperimentsInHonesty** [comment]
@agosmou Hi.  Just reminding you, this issue needs you to self assign and choose your role label, etc.

**Comment by agosmou** [comment]
Hi Bonnie! In progress! :)

**Comment by ExperimentsInHonesty** [comment]
@agosmou Thanks!  I've moved this to in progress.

**Comment by agosmou** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

About 2 months (only b/c I was out of town for the majority of this time, so I just picked up where I left off after onboarding)

**Comment by agosmou** [comment]
1. Progress: Picked up where I had left off - I just need to revisit steps 1.1-1.6 since I had done most of this a couple months back
2. Blockers: Waiting to be re-added to the calendar invites and the google drive (corresponding with Justin Dingeman currently). 
3. Availability: I can finish this by EOD 3/7/23
4. ETA: 3/7/23

**Comment by ExperimentsInHonesty** [comment]
@agosmou I see that you got re-added to the calendar invites. 

I noticed that you have not picked up a second issue.  It's possible when you looked for one, there was not one available.  

There is 1 available now [Good front end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257).  

If you don't find one when you look again, go ahead and do another [good first issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257).  They are basicly the same type of issue, we just need people to do two total so that we can make sure you dev environment is setup and you know how to do branching effectively.

**Comment by jdingeman** [comment]
Dev is currently traveling in Europe and informed me he plans on picking back up upon his return April 10, 2024.

**Comment by agosmou** [comment]
Just returned from travel.

Picked up another good first issue today, per Bonnie's direction above.

**Comment by agosmou** [comment]
As of 4/17/23, I've completed two, 2, "good-first-issue" items.

Waiting for an update on the priority backlog for front end "small-issues"

**Comment by agosmou** [comment]
My small issue was review, approved, and merged last week.

Awaiting a "medium" issue on the priority backlog.

**Comment by agosmou** [comment]
Picked up a medium issue - currently working through it.

**Comment by agosmou** [comment]
The medium issue was successfully merged - awaiting a large issue to be moved to the priority backlog so I can pick it up.

**Comment by agosmou** [comment]
Picked up a large issue

**Comment by agosmou** [comment]
Awaiting merge on large issue

**Comment by agosmou** [comment]
https://github.com/hackforla/website/issues/4593

Large issue successfully merged

**Comment by agosmou** [comment]
Also created an issue just in case this is needed as part of onboarding - pending dev leads approval. https://github.com/hackforla/website/issues/4699

Officially completed pre-work checklist.

**Comment by roslynwythe** [comment]
Hi @agosmou thank you for your great work so far with Hack for LA.  We are undertaking some changes to the pre-work checklists and as a result, they will remain open much longer, and possibly indefinately, and they will be used for tracking purposes.  We will be introducing some automation so that information about your other issues is incorportated into this issue, but in the meantime, we are asking devs to create a progress update in this issue at least every two weeks, and to remove the '2 week inactive' label.  Thank you

**Comment by agosmou** [comment]
Completed

**Comment by roslynwythe** [comment]
Hi @agosmou.  Congratulations on completing your `Complexity: Large` issue.  We really appreciate your contributions to Hack for LA.  During periods when you are not working on any issues, we request that you post a comment in this issue to inform us of your progress, any problems you might be having, and to make sure that you are still interested in continuing with Hack for LA.  We would like to see updates,in this issue or in other issues, at least every two weeks.  Thank you

**Comment by agosmou** [comment]
Currently collaborating on TDM and HUU as well as the Engineering CoP

**Comment by agosmou** [comment]
Working on HUU, TDM, and Eng CoP

**Comment by agosmou** [comment]
Still volunteering!

**Comment by HackforLABot** [comment]
## Activity Log: agosmou
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- agosmou opened: https://github.com/hackforla/website/issues/3787 at 01/03/2023, 08:47 PM PST
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1427628560 at 02/13/2023, 01:41 AM PST
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1432358642 at 02/15/2023, 05:56 PM PST
- agosmou assigned: https://github.com/hackforla/website/issues/3787#issuecomment-1437772588 at 02/20/2023, 08:36 PM PST
- agosmou unassigned: https://github.com/hackforla/website/issues/3787#issuecomment-1437772588 at 02/20/2023, 08:37 PM PST
- agosmou assigned: https://github.com/hackforla/website/issues/3787#issuecomment-1437772588 at 02/20/2023, 08:37 PM PST
- agosmou unassigned: https://github.com/hackforla/website/issues/3787#issuecomment-1437772588 at 02/20/2023, 08:38 PM PST
- agosmou assigned: https://github.com/hackforla/website/issues/3787#issuecomment-1437772588 at 02/20/2023, 08:38 PM PST
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1437874914 at 02/20/2023, 09:17 PM PST
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1455739090 at 03/06/2023, 01:02 AM PST
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1455745973 at 03/06/2023, 01:06 AM PST
- agosmou assigned: https://github.com/hackforla/website/issues/3987 at 03/14/2023, 06:26 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3987#issuecomment-1469155989 at 03/14/2023, 06:44 PM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4191 at 03/14/2023, 08:03 PM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4191#event-8789834228 at 03/19/2023, 08:28 PM PDT
- agosmou issue closed by PR 4191: https://github.com/hackforla/website/issues/3987#event-8789834340 at 03/19/2023, 08:28 PM PDT
- agosmou assigned: https://github.com/hackforla/website/issues/4450 at 04/14/2023, 02:19 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1509373629 at 04/14/2023, 03:58 PM PDT
- agosmou unassigned: https://github.com/hackforla/website/issues/4450 at 04/14/2023, 04:03 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4450#issuecomment-1509376520 at 04/14/2023, 04:03 PM PDT
- agosmou assigned: https://github.com/hackforla/website/issues/4382 at 04/14/2023, 04:04 PM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4519 at 04/14/2023, 04:37 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4382#issuecomment-1509400329 at 04/14/2023, 04:49 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4519#issuecomment-1509498707 at 04/14/2023, 09:31 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4519#issuecomment-1509989606 at 04/15/2023, 03:23 PM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4519#event-9029423084 at 04/17/2023, 02:46 PM PDT
- agosmou issue closed by PR 4519: https://github.com/hackforla/website/issues/4382#event-9029423241 at 04/17/2023, 02:46 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1512243501 at 04/17/2023, 04:46 PM PDT
- agosmou assigned: https://github.com/hackforla/website/issues/4450 at 04/18/2023, 08:30 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4450#issuecomment-1514082577 at 04/18/2023, 08:31 PM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4548 at 04/19/2023, 12:09 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4450#issuecomment-1515246447 at 04/19/2023, 12:16 PM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4548#event-9060817451 at 04/20/2023, 12:28 PM PDT
- agosmou issue closed by PR 4548: https://github.com/hackforla/website/issues/4450#event-9060817611 at 04/20/2023, 12:28 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1519203754 at 04/23/2023, 05:08 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1523895984 at 04/26/2023, 11:53 AM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4573 at 04/27/2023, 01:43 AM PDT
- agosmou submitted review: https://github.com/hackforla/website/pull/4569#pullrequestreview-1403544518 at 04/27/2023, 01:59 AM PDT
- agosmou submitted review: https://github.com/hackforla/website/pull/4570#pullrequestreview-1403551816 at 04/27/2023, 02:01 AM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1527931681 at 04/28/2023, 11:21 AM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1527937801 at 04/28/2023, 11:27 AM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1528020138 at 04/28/2023, 12:53 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1528125064 at 04/28/2023, 02:45 PM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4573#event-9141592994 at 05/01/2023, 09:19 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1530950659 at 05/01/2023, 11:27 PM PDT
- agosmou assigned: https://github.com/hackforla/website/issues/4593 at 05/07/2023, 10:51 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4593#issuecomment-1537790486 at 05/07/2023, 10:53 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1537796637 at 05/07/2023, 11:00 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4593#issuecomment-1549050520 at 05/15/2023, 11:11 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4593#issuecomment-1552364944 at 05/17/2023, 08:53 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4593#issuecomment-1553519289 at 05/18/2023, 12:12 PM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4694 at 05/18/2023, 02:26 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1553671300 at 05/18/2023, 02:31 PM PDT
- agosmou opened: https://github.com/hackforla/website/issues/4699 at 05/18/2023, 10:52 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4694#issuecomment-1556079615 at 05/20/2023, 09:25 PM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4694#event-9295443462 at 05/20/2023, 11:15 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1556334390 at 05/21/2023, 05:00 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1556335233 at 05/21/2023, 05:02 PM PDT
- agosmou assigned: https://github.com/hackforla/website/issues/4753 at 05/29/2023, 09:58 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/4753#issuecomment-1567755228 at 05/29/2023, 09:59 PM PDT
- agosmou opened: https://github.com/hackforla/website/pull/4760 at 05/30/2023, 10:35 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4760#issuecomment-1571329647 at 05/31/2023, 09:43 PM PDT
- agosmou commented on: https://github.com/hackforla/website/pull/4760#issuecomment-1572204493 at 06/01/2023, 07:50 AM PDT
- agosmou PR was merged: https://github.com/hackforla/website/pull/4760#event-9407247915 at 06/01/2023, 12:40 PM PDT
- agosmou issue closed by PR 4760: https://github.com/hackforla/website/issues/4753#event-9407248120 at 06/01/2023, 12:40 PM PDT
- agosmou opened: https://github.com/hackforla/website/issues/4769 at 06/01/2023, 08:27 PM PDT
- agosmou opened: https://github.com/hackforla/website/issues/4770 at 06/01/2023, 08:35 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1588242939 at 06/12/2023, 04:25 PM PDT
- agosmou commented on: https://github.com/hackforla/website/issues/3787#issuecomment-1622571219 at 07/05/2023, 02:47 PM PDT
- agosmou unassigned: https://github.com/hackforla/website/issues/4593 at 02/20/2024, 03:39 PM PST
